collect_app/src/main/java/org/odk/collect/android/widgets/VideoWidget.java

```diff
@@ -352,26 +285,11 @@ public void denied() {
     private void captureVideo() {
         Intent i;
         if (selfie) {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-                i = new Intent(getContext(), CaptureSelfieVideoActivityNewApi.class);
-            } else {
-                i = new Intent(getContext(), CaptureSelfieVideoActivity.class);
```

Comment:
The same here, looks as if this class was used just here so we can get rid of it.